test/suites/integration/ami_test.go

```diff
@@ -189,6 +189,20 @@ var _ = Describe("AMI", func() {
 			}})
 			provisioner := test.Provisioner(test.ProvisionerOptions{
 				ProviderRef: &v1alpha5.MachineTemplateRef{Name: provider.Name},
+				// TODO: remove requirements after Ubuntu fixes bootstrap script issue w/
+				// new instance types not included in the max-pods.txt file.
```

Comment:
nit: Is there an upstream issue somewhere else we can add in this comment so future authors know when to remove this?

Reply:
nope :( the Ubuntu bootstrap script isn't open sourced :(